mscore/musescore.cpp

```diff
@@ -7473,21 +7473,21 @@ MuseScoreApplication::CommandLineParseResult MuseScoreApplication::parseCommandL
-      parser.addOption(QCommandLineOption("run-test-script", "Run script tests listed in the command line arguments"));
+      parser.addOption(QCommandLineOption(      "run-test-script", "Run script tests listed in the command line arguments"));
       parser.addOption(QCommandLineOption({"M", "midi-operations"}, "Specify MIDI import operations file", "file"));
       parser.addOption(QCommandLineOption({"w", "no-webview"}, "No web view in start center"));
       parser.addOption(QCommandLineOption({"P", "export-score-parts"}, "Used with '-o <file>.pdf', export score and parts"));
       parser.addOption(QCommandLineOption(      "no-fallback-font", "Don't use Bravura as fallback musical font"));
       parser.addOption(QCommandLineOption({"f", "force"}, "Used with '-o <file>', ignore warnings reg. score being corrupted or from wrong version"));
       parser.addOption(QCommandLineOption({"b", "bitrate"}, "Used with '-o <file>.mp3', sets bitrate, in kbps", "bitrate"));
       parser.addOption(QCommandLineOption({"E", "install-extension"}, "Install an extension, load soundfont as default unless if -e is passed too", "extension file"));
-      parser.addOption(QCommandLineOption("score-media", "Export all media (excepting mp3) for a given score in a single JSON file and print it to std out"));
-      parser.addOption(QCommandLineOption("score-meta", "Export score metadata to JSON document and print it to stdout"));
-      parser.addOption(QCommandLineOption("score-mp3", "Generates mp3 for the given score and export the data to a single JSON file, print it to std out"));
-      parser.addOption(QCommandLineOption("score-parts-pdf", "Generates parts data for the given score and export the data to a single JSON file, print it to std out"));
-      parser.addOption(QCommandLineOption("score-transpose", "Transposes the given score and exports the data to a single JSON file, prints it to std out", "options"));
-      parser.addOption(QCommandLineOption("raw-diff", "Print a raw diff for the given scores"));
-      parser.addOption(QCommandLineOption("diff", "Print a diff for the given scores"));
+      parser.addOption(QCommandLineOption(      "score-media", "Export all media (excepting mp3) for a given score in a single JSON file and print it to stdout"));
+      parser.addOption(QCommandLineOption(      "score-meta", "Export score metadata to JSON document and print it to stdout"));
+      parser.addOption(QCommandLineOption(      "score-mp3", "Generates mp3 for the given score and export the data to a single JSON file, print it to stdout"));
+      parser.addOption(QCommandLineOption(      "score-parts-pdf", "Generates parts data for the given score and export the data to a single JSON file, print it to stdout"));
+      parser.addOption(QCommandLineOption(      "score-transpose", "Transposes the given score and exports the data to a single JSON file, print it to stdout", "options"));
```

Comment:
`prints` because all other verbs are also in this form

Reply:
I see "Print...", "Use...", "Start...", "Dump...", "Export...", "Set...", "Specify", "Load...", "Execute...", "Save...", "Revert...", "Process...", "Enable...", "Override...", "Run...", "Don't use...", "Install...".
So the majority is not, but "Used with...", "Generates...", "Transposes", so needs fixing.